# Torque on/off

string cmd_type
string name
bool enable
---